.github/workflows/tasklist-ci.yml

```diff
@@ -49,12 +74,170 @@ jobs:
+    runs-on: ubuntu-latest
+    needs: check_changes
+    if: ${{ needs.check_changes.outputs.has_changed_frontend == 'true' }}
+    container:
+      image: mcr.microsoft.com/playwright:v1.43.1
+      options: --user 1001:1000
+    defaults:
+      run:
+        working-directory: tasklist/client
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@1d96c772d19495a3b5c517cd2bc0cb401ea0529f # v4
+      - name: Setup yarn cache
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: "yarn"
+          cache-dependency-path: tasklist/client/yarn.lock
+      - name: Install node dependencies
+        run: yarn
+      - name: Build frontend
+        run: yarn build:visual-regression
+      - name: Start server
+        run: yarn start:visual-regression &
+      - name: Run Playwright tests
+        run: yarn playwright e2e/visual
+      - uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: visual-regression-report
+          path: tasklist/client/playwright-report/
+          retention-days: 30
+
+  fe-a11y-tests:
+    name: a11y tests
+    runs-on: ubuntu-latest
+    needs: check_changes
+    if: ${{ needs.check_changes.outputs.has_changed_frontend == 'true' }}
+    container:
+      image: mcr.microsoft.com/playwright:v1.43.1
+      options: --user 1001:1000
+    defaults:
+      run:
+        working-directory: tasklist/client
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@1d96c772d19495a3b5c517cd2bc0cb401ea0529f # v4
+      - name: Setup yarn cache
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: "yarn"
+          cache-dependency-path: tasklist/client/yarn.lock
+      - name: Install node dependencies
+        run: yarn
+      - name: Build frontend
+        run: yarn build
+      - name: Start server
+        run: yarn start:visual-regression &
+      - name: Run A11y tests
+        run: yarn playwright a11y
+      - uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: a11y-report
+          path: tasklist/client/playwright-report/
+          retention-days: 30
+
   test-summary:
     # Used by the merge queue as a dummy instead of the Zeebe CI in case only Operate files got changed:
     # https://github.com/orgs/community/discussions/26251
     # This name is hard-coded in the branch rules; remember to update that if this name changes
     name: Test summary
     if: always()
     runs-on: ubuntu-latest
+    needs:
+      - check_changes
+      - fe-type-check
+      - fe-eslint
+      - fe-tests
+      - fe-visual-regression-tests
+      - fe-a11y-tests
+      - fe-stylelint
     steps:
-      - run: exit 0
+      - run: exit ${{ ((needs.check_changes.outputs.has_changed_frontend == 'false' && contains(needs.*.result, 'skipped') || contains(needs.*.result, 'failure')) && 1) || 0 }}
```

Comment:
I think this should be
`((needs.check_changes.outputs.has_changed_frontend == 'true' && contains(needs.*.result, 'skipped')`
as you want to return failed summary (status 1_ when "mandatory" jobs were skipped or failed

Reply:
@houssain-barouni That's why I think it might be better to split the frontend and backend CI for required jobs, because then we won't be able to properly test for skipped jobs when we add other backend jobs, it's just easier to use the workflow trigger paths